rest-api/src/main/resources/definitions/study.yml

```diff
@@ -205,10 +205,8 @@ properties:
         x-nullable: false
     reauthenticationEnabled:
         type: boolean
-        default: false
         description: |
             **Can only be set by an administrator.** Enable or disable the ability to issue a reauthentication token in a session, and call the reauthentication APIs to issue a new session. 
-        x-nullable: false
```

Comment:
I believe default = true, and x-nullable = false here, but only if we prevent the user from nulling out the value on an update. Since the boolean was a primitive before, no client will be expecting this property to be null (and apparently this matters in iOS, which is why we added the x-nullable information).